uint64 obj_id
Header header
uint64 framenumber
float32[4] state_vec
uint8 is_living
